### PR TITLE
Add travis to the project (issue #3)

### PR DESCRIPTION
**Pull request by nctl144**
I added Travis to the project. Let me know if there is anything that I can improve! #3

### CODE REVIEW

**Comment by lopuhin** [comment]
Thanks @nctl144 !

Ideally, it would be better to debug the build in the branch, as with travis you can never be sure it's correct before the build passes. I just activated travis-ci for this repo, so I hope if you push anything to your branch, the first build will start - could you please try it?

Another issue is that right now you call ``tox bookworm``, I'm not sure it will work on travis in it's current state, as it requires a books.toscrape running, right? Although maybe I'm missing something. But if this is right, maybe it makes sense to start with making something simpler to run, like cssbench, linkextractor and xpathbench benchmarks which do not require external dependencies AFAIK.

**Comment by nctl144** [comment]
Hey @lopuhin , indeed bookworm and broadcrawl need a local server to run. So I will try to create Travis tests for other commands as you suggested

**Comment by nctl144** [comment]
I have added to travis to run 3 commands cssbench, xpathbench and link! Please let me know if there is anything I can improve.

File: tox.ini [hunk 1]
```diff
@@ -15,4 +15,7 @@ deps=
     statistics
     six
 
-commands= python bench.py --n-runs 2 --upload_result {posargs}
```
Comment (lopuhin) [inline tox.ini]: Right now ``tox`` command can be used to run any benchmarks, but after this change only specific benchmarks will be run. I agree that current default ``tox`` command is not great for running on travis due to ``--n-runs 2`` and especially ``--upload_result``, so maybe it makes sense to leave just ``python bench.py {posargs}`` here, and call all benchmarks we want to run one by one from travis?

Even better would be if by default, all benchmarks would be run with a single tox command, and individual benchmarks could be specified optionally, but this is probably out of scope for this PR.

Comment (nctl144) [reply tox.ini]: I made a new commit to call the tests from Travis. But I like the idea of running the tests with a single tox command. Can you elaborate more on how you want to do that? :)

**Comment by nctl144** [comment]
Oops I'm sorry for the mess. I accidentally did a rebase instead of merge :/ I will pay attention to it next time!

**Comment by lopuhin** [comment]
> Oops I'm sorry for the mess. I accidentally did a rebase instead of merge :/ I will pay attention to it next time!

Actually rebase is totally fine I think, unless you made another branch off that branch, then rebasing will complicate merging changes into that new branch.

File: tox.ini (hunk 1 above)
Comment (lopuhin) [reply tox.ini]: Thanks, I'll check it a bit later! 

> I like the idea of running the tests with a single tox command. Can you elaborate more on how you want to do that? :)

Not quite sure, I never used click myself. With argparse, I would probably make bench.py accept multiple positional arguments (names of the benchmarks to run), and make it default to all benchmarks. Alternatively, default value can live in tox.ini, but still bench.py must accept multiple benchmark names in this case.

Comment (nctl144) [reply tox.ini]: With my new commit, I think I made it :) I tested it and the bench.py now can take multiple commands and we can call tox once.

**Pull request merged**